src/main/java/commission/mapper/PeopleRowMapper.java

```diff
@@ -0,0 +1,17 @@
+package commission.mapper;
+
+import commission.entity.Person;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class PeopleRowMapper implements RowMapper<Person> {
+    @Override
+    public Person mapRow(ResultSet rs, int rowNum) throws SQLException {
+        return new Person(rs.getLong(1),
+                rs.getString(2),
+                rs.getString(3));
```

Comment:
Just a bit of a style thing here, check your intellij settings for indents and make sure everything is at 4. You don't want to have double indentation as then there is too much white space. Also I will give a suggestion on my personal preference for splitting code across lines like this.

```suggestion
        return new Person(
            rs.getLong(1),
            rs.getString(2),
            rs.getString(3)
        );
```

Reply:
Got it. It's much easier to read.